apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -624,7 +626,7 @@ void wm_auto_test(void *arg)
 {
```

Comment:
@pillip8282 I totally agree with your comment. As I mentioned above, a new command to measure code coverage instead of auto test will be updated soon. In auto_test, the function will be immediately terminated with error log, when it fails.